source/operations/concepts.rst

```diff
@@ -186,7 +186,12 @@ Some common reasons for bit rot include:
 - accidental overwrites
 
 MinIO uses a hashing algorithm to confirm the integrity of an object.
-If an object becomes corrupted by bit rot, MinIO can automatically heals the object depending on the system topology and availability.
+This algorithm automatically applies at the time of any ``GET`` request that calls an object.
+If an object becomes corrupted by bit rot, MinIO can automatically :ref:`heal <minio-concepts-healing>` the object depending on the availability of parity shards for the object.
```

Comment:
`can automatically heal` suggests this behavior is optional? Perhaps `automatically heals` would work.

Reply:
I am somewhat wondering about scanner healing vs `GET` read time healing. Are those the same thing, but done at different times?

Reply:
The healing itself is similar. Bit rot healing is on that specific object.
The scanner checks a selection of objects with each pass, but doesn't check for bit rot unless that is specifically enabled (with a significant performance hit as a result).